packages/flutter/lib/src/material/colors.dart

```diff
@@ -10,6 +10,76 @@ import 'package:flutter/painting.dart';
 /// darker the color. There are 10 valid indices: 50, 100, 200, ..., 900.
 /// The value of this color should the same the value of index 500 and [shade500].
 ///
+/// ## Updating to [ColorScheme]
+///
+/// There is a new class,[ColorScheme], that is is preferred for
```

Comment:
ColorScheme isn't really "new" and characterizing it that way will not age well. Better to just say The [ColorScheme] class is the preferred way to configure the colors for Material 3 applications.

Reply:
Fixed!